package-lock.json

```diff
@@ -0,0 +1,9797 @@
+{
```

Comment:
I also prefer `yarn`, but that's out of scope for this PR.

Reply:
IMO you should just remove `package-lock.json`

Reply:
![Alt Text](https://media.giphy.com/media/1M9fmo1WAFVK0/giphy.gif)
Lock files (whether npm's `package-lock.json` or yarn's `yarn.lock`) speed up your package dependency installation time and guarantee a predictable, known-to-work dependency tree. 

Why wouldn't we want to have those advantages for free?

Reply:
I don’t know about the newest NPM but a year ago I had many issues with `package-lock.json` - it was unpredictable, installing different versions for the same file and often causing a failure during installation

Reply:
Sorry to hear about your negative experience with npm :(. Yeah, versions 4 and 5 were rushed (due to competition from yarn) and had plenty of bugs. In fact, some are [still open](https://github.com/npm/npm/issues/17722)...

From my experience, the latest version (6) is more robust and works quite well. I actually recently switched the project I'm leading to yarn not because of npm (although that too), but because yarn has awesome features which npm does not.

Bottom line: I suggest we keep package-lock.json and consider moving to yarn in a separate issue/PR.

Reply:
we can try that

Reply:
@elektronik2k5 We'll keep `package-lock.json` for now and consider moving to Yarn after some research if the features it provides would suit this package. 

I personally didn't have any problems with `npm` :)